src/sage/matroids/database/driver_functions.py

```diff
@@ -0,0 +1,165 @@
+r"""
+Functions that access the collections of matroids
+
+This module contains driver functions to easily access the collections of
+matroids in the database. Each of these functions returns a complete list
+of matroids from the corresponding collection. These functions can be viewed
+by typing ``matroids.`` and hitting the ``tab`` button.
+
+AUTHORS:
+- Giorgos Mousa (2023-12-08): initial version
+
+Functions
+=========
+
+"""
+# **********************************************************************
+#       Copyright (C) 2023 Giorgos Mousa <gmousa@proton.me>
+#
+#  Distributed under the terms of the GNU General Public License (GPL)
+#  as published by the Free Software Foundation; either version 2 of
+#  the License, or (at your option) any later version.
+#                  https://www.gnu.org/licenses/
+# **********************************************************************
+
+
+def OxleyMatroids():
+    """
+    Return a list of (nonparameterized) matroids
+
+    As listed in
+    :mod:`matroids.database.oxley_matroids <sage.matroids.database.oxley_matroids>`,
+    following the Appendix ``Some Interesting Matroids`` in [Oxl2011]_ (p.
+    639-64).
+
+    EXAMPLES::
+
+        sage: len(matroids.OxleyMatroids())
+        41
+
+    """
+    Matroids = []
+    from sage.matroids.database.oxley_matroids import (
+        U24, U25, U35, K4, Whirl3, Q6, P6, U36, R6,
+        Fano, FanoDual, NonFano, NonFanoDual, O7, P7,
+        AG32, AG32prime, R8, F8, Q8, L8, S8, Vamos, T8, J, P8, P8pp,
+        Wheel4, Whirl4,
+        K33dual, K33, AG23, TernaryDowling3, Pappus, NonPappus,
+        K5, K5dual, R10,  # NonDesargues,
+        R12,  # S_5_6_12,
+        T12,
+        PG23
+    )
+
+    lst = {
+        4: [U24],
+        5: [U25, U35],
+        6: [K4, Whirl3, Q6, P6, U36, R6],
+        7: [Fano, FanoDual, NonFano, NonFanoDual, O7, P7],
+        8: [
+            AG32, AG32prime,
+            R8, F8, Q8, L8, S8,
+            Vamos, T8, J, P8, P8pp,
+            Wheel4, Whirl4
+        ],
+        9: [K33dual, K33, AG23, TernaryDowling3, Pappus, NonPappus],
+        10: [K5, K5dual, R10],
+        12: [R12, T12],
+        13: [PG23],
+    }
+    for i in lst:
+        for M in lst[i]:
+            Matroids.append(M())
+    return Matroids
+
+
+def BrettellMatroids():
+    """
+    Return a list of interesting matroids
```

Comment:
Are the others uninteresting? I think you should be more consistent with the one-line docstrings of these functions.